src/checkers/inference/solver/constraintgraph/GraphBuilder.java

```diff
@@ -132,8 +134,10 @@ private void calculateConstantPath() {
                             continue;
                         } else {
                             if (InferenceMain.getInstance().getVisitor() instanceof DataflowVisitor) {
```

Comment:
Should we add a TODO reminder to get rid of this special handling in the future?

Reply:
TODO added